opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -44,6 +44,33 @@ message InstrumentationLibraryMetrics {
   repeated Metric metrics = 2;
 }
 
+// A representation of raw measurements, which can have statistical meaning based
+// on how the measurement was sampled, as well as a span ID and trace ID of the
+// active span when the measurement was recorded
+message RawValue {
+  // The set of labels that were dropped by the aggregator, but recorded
+  // alongside the original measurement. Only labels that were dropped by the aggregator should be included
+  repeated opentelemetry.proto.common.v1.StringKeyValue labels = 1;
```

Comment:
What is the relationship between these labels and the labels in the DataPoint:
* Do we duplicate them?
* Do we extract these labels and the actual set of labels is the combination of these + datapoint.lables?

Reply:
For exemplars these labels are only the labels _not_ included in the DataPoint's labels. I would change this field to be `dropped_labels` but if RawValue is used as a data point itself the labels would include all labels

Reply:
@jmacd I know you tried to share the messages, can you help define the behavior here?